### ISSUE TITLE
Crash when closing Options

### ISSUE THREAD

**Issue by rqueizan** [winsiderss/systeminformer#1821]
### Brief description of your issue

When showing the settings window and closing it on the close button, an error occurs.

### Steps to reproduce (optional)

Show main windows of System Informar.
Click on Options.
I move from the first option to the last one using the down key
Click on Close button.

### Expected behavior (optional)

The options window should close and return to the main window.

### Actual behavior (optional)

The application crash and show this window:
![image](https://github.com/winsiderss/systeminformer/assets/18214796/4174000a-0967-4593-8943-363c10304314)

### Environment (optional)

```shell
System Informer 3.0.7029 (f647fb0)
Windows 11 Enterprise 22H2 Build 22621.2134
AMD Ryzen 5 4600H with Radeon Graphics
```

**Comment by rqueizan** [comment]
I made a Fulldump, if you need it I'll upload it.

**Comment by jxy-s** [comment]
Probably is inside the device notification options. I'll correct this shortly.

**Comment by jxy-s** [comment]
Should be fixed with https://github.com/winsiderss/systeminformer/commit/d137ee88b70912855cd9e400a9d9601b83eda979 in the next build

**Comment by rqueizan** [comment]
Solved in Version: 3.0.7039 (1dc68ca)
